pywren/ec2standalone.py

```diff
@@ -173,10 +173,10 @@ def terminate_instances(instance_list):
     # FIXME delete individuals
     """
     for instance_name, instance_obj in instance_list:
+        logger.debug('Terminating instance %s', instance_name)
```

Comment:
No its actually trying to use `instance_name` to avoid an unused variable warning. i can also make it an `_`